src/NzbDrone.Core/Messaging/Commands/CommandExecutor.cs

```diff
@@ -120,47 +62,69 @@ private dynamic GetCommand(string commandTypeName)
 
             try
             {
-                _trackCommands.Start(command);
-                BroadcastCommandUpdate(command);
+                _commandQueueManager.Start(commandModel);
+                BroadcastCommandUpdate(commandModel);
 
                 if (!MappedDiagnosticsContext.Contains("CommandId") && command.SendUpdatesToClient)
```

Comment:
we should set this regardless of `SendUpdateToClient` we can use it later for grouping log messages